Include/internal/pycore_dtoa.h

```diff
@@ -1,4 +1,6 @@
 #ifndef PY_NO_SHORT_FLOAT_REPR
+#ifndef Py_CORE_DTOA_H
```

Comment:
To follow other files, you should use "INTERNAL" instead: `#ifndef Py_INTERNAL_DTOA_H`.